srcpkgs/wvkbd/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'wvkbd'
+pkgname=wvkbd
+version=0.7
+revision=1
+build_style=gnu-makefile
+make_build_args="CC=$CC"
+hostmakedepends="wayland-devel pkg-config"
+makedepends="pango-devel cairo-devel libxkbcommon-devel"
```

Comment:
wayland-devel needs to be in both hostmakedepends and makedepends. Did you try cross-building this locally?

Reply:
Thanks, I cross-build it to x86_64-musl, i686, armv7l, armv7l-musl, armv6l, armvl6-musl, aarch64, arch64-musl